macros/CSS_Ref.ejs

```diff
@@ -471,7 +471,7 @@ for (var i in characters) {
             case "translateX()":
             case "translateY()":
             case "translateZ()":
-                item.urlPath = "transform-function#" + item.label;
+                item.urlPath = "transform-function/" + item.label.replace(/\(\)/, "");
```

Comment:
Is there a reason this must be a regex instead of just `.replace("()", "")`?

Reply:
Yes, you're right. Change made.